cinema/tests/test_movie_and_movie_session.py

```diff
@@ -0,0 +1,246 @@
+                )
+            )
+        )
+
+        movie_session_serializer = MovieSessionListSerializer(movie_sessions, many=True)
+
+        self.assertEqual(movie_sessions_response.status_code, status.HTTP_200_OK)
+        self.assertEqual(movie_session_serializer.data, movie_sessions_response.data)
+
+    def test_filter_movie_by_title(self):
+        movie_response_apalon = self.client.get(MOVIE_LIST_URL, {"title": "apalon"})
+        movie_response_vinston = self.client.get(MOVIE_LIST_URL, {"title": "vinston"})
+
+        movie_serializer_apalon = MovieListSerializer(self.movie_apalon, many=False)
+        movie_serializer_vinston = MovieListSerializer(self.movie_vinston, many=False)
+
+        self.assertIn(movie_serializer_apalon.data, movie_response_apalon.data)
+        self.assertIn(movie_serializer_vinston.data, movie_response_vinston.data)
+
+        self.assertNotIn(movie_serializer_vinston.data, movie_response_apalon.data)
+        self.assertNotIn(movie_serializer_apalon.data, movie_response_vinston.data)
+
+    def test_filter_movie_by_actors(self):
+        movie_responce_Illia = self.client.get(
+            MOVIE_LIST_URL, {"actors": f"{self.actor_Illia.id}"}
+        )
+        movie_responce_Jonh = self.client.get(
+            MOVIE_LIST_URL, {"actors": f"{self.actor_John.id}"}
+        )
+
+        movie_serializer_Jonh = MovieListSerializer(self.movie_apalon, many=False)
+        movie_serializer_Illia = MovieListSerializer(self.movie_vinston, many=False)
+
+        self.assertEqual(
+            movie_serializer_Illia.data["actors"],
+            movie_responce_Illia.data[0]["actors"],
+        )
+        self.assertEqual(
+            movie_serializer_Jonh.data["actors"], movie_responce_Jonh.data[0]["actors"]
+        )
+
+        self.assertNotEqual(
+            movie_serializer_Jonh.data["actors"], movie_responce_Illia.data[0]["actors"]
+        )
+
+    def test_filter_movie_by_genres(self):
+        movie_response_horror = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"{self.genre_horror.id}"}
+        )
+        movie_response_drama = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"{self.genre_drama.id}"}
+        )
+
+        movie_serializer_horror = MovieListSerializer(self.movie_apalon, many=False)
+        movie_serializer_drama = MovieListSerializer(self.movie_vinston, many=False)
+
+        self.assertEqual(
+            movie_serializer_horror.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+        self.assertEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_drama.data[0]["genres"],
+        )
+        self.assertNotEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+
+    def test_filter_movie_session_by_date(self):
+        movie_session_response_2023 = self.client.get(
+            MOVIE_SESSION_LIST_URL,
```

Comment:
The same applies here. Add a test case for a date that does not exist in the database.